apps/web/src/app/api/projects/[projectId]/commits/[commitUuid]/documents/route.ts

```diff
@@ -7,8 +7,9 @@ import { authHandler } from '$/middlewares/authHandler'
 import { errorHandler } from '$/middlewares/errorHandler'
 import { NextRequest, NextResponse } from 'next/server'
 
-export const GET = errorHandler(
-  authHandler(
+type IParam = { projectId: number; commitUuid: string }
+export const GET = errorHandler<IParam, DocumentVersion[]>(
+  authHandler<IParam, DocumentVersion[]>(
```

Comment:
I wonder if we could put under `authHandler` the `errorHandler`. This looks super repetitive